build.rs

```diff
@@ -7,15 +7,15 @@ fn main() {
     let status = Command::new("make")
         .arg("clean")
         .arg("-C")
-        .arg(format!("{}/python/grizzly/", project_dir))
+        .arg(format!("{}/python/grizzly/grizzly", project_dir))
         .status()
         .unwrap();
     assert!(status.success());
 
     let status = Command::new("make")
         .arg("convertor")
         .arg("-C")
-        .arg(format!("{}/python/grizzly/", project_dir))
+        .arg(format!("{}/python/grizzly/grizzly", project_dir))
```

Comment:
We shouldn't be making the convertor dylib as part of the weld installation step. Also since we're not invoking setup.py the Makefile will use the default "which python" function which may be different than the python you run "python setup.py install". I vote to just delete these lines.

Reply:
Agreed, let's remove this -- `setup.py` calls `make convertor` now, right?